backend/tests/newman/gotCaked.postman_collection.json

```diff
@@ -2572,6 +2688,137 @@
 					},
 					"response": []
 				},
+				{
+					"name": "Add Picture to Recipe",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Status Code is 200\", function() {\r",
+									"    pm.response.to.have.status(200);\r",
+									"});\r",
+									"\r",
+									"var data = JSON.parse(responseBody);\r",
+									"\r",
+									"pm.test(\"Recieving Correct Recipe\", function () {\r",
+									"    pm.expect(data.user_id).equal(pm.globals.get(\"userID-3\"));\r",
+									"    pm.expect(data.id).equal(pm.globals.get(\"recipeId-2\"));\r",
+									"    pm.expect(data.picture_uri[0]).exist;    \r",
+									"    pm.globals.set(\"picture_uri\", data.picture_uri[0]);\r",
+									"}); "
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "prerequest",
+							"script": {
+								"exec": [
+									"let userName = pm.variables.replaceIn(\"{{$randomUserName}}\");\r",
+									"pm.globals.set(\"randomUserNEW\", userName);"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "PATCH",
+						"header": [
+							{
+								"key": "jwt",
+								"value": "{{jwtToken-3}}",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "formdata",
+							"formdata": [
+								{
+									"key": "picture",
+									"type": "file",
+									"src": "/C:/Users/HUIDAL/Nextcloud/3. Semester/Webengineering 2/WebDev-Projekt/Icons and Pictures/russischer-zupfkuchen.jpg"
```

Comment:
Please use the working directory feature to make testing easier on other devices.

Reply:
https://learning.postman.com/docs/getting-started/settings/

Reply:
Thx, I had completely forgotten this issue :D

Reply:
Is fixed now